src/cljs_time/coerce.cljs

```diff
@@ -50,7 +50,7 @@
 (defn to-epoch
   "Convert `obj` to Unix epoch."
   [obj]
-  (some-> obj to-long (/ 1000)))
+  (some-> obj to-long (/ 1000) int))
```

Comment:
Makes sense to me. Might want to quickly check perf implications, though I doubt there would be a problem as it drops down to native JS mod operator.